dev-requirements.txt

```
@@ -19,3 +19,9 @@ black>=22.8,<22.9
 setuptools>56
 # Debuggery
 icecream>=2.1
+# typing
+mypy>=0.942
```

```suggestion
mypy==0.991
```

Better to pin to a specific version and let dependabot bump it. Any new version of mypy could result in new errors appearing and breaking the CI (e.g. because new features reveal errors that were uncaught in previous versions).